### PR TITLE
Add support for object in length built-in filter

### PR DESCRIPTION
**Pull request by KoltesDigital**
Discussed in #480.

### CODE REVIEW

File: docs/content/docs/_index.md [hunk 1]
```diff
@@ -811,7 +811,7 @@ Example: `{{ value | join(sep=" // ") }}`
 If value is the array `['a', 'b', 'c']`, the output will be the string "a // b // c".
 
 #### length
-Returns the length of an array or a string, 0 if the value is not an array.
+Returns the length of an array, an object, or a string. Always 0 for other types.
```
Comment (Keats) [inline docs/content/docs/_index.md]: Can you change the other types to error instead?

Comment (KoltesDigital) [reply docs/content/docs/_index.md]: Done. The comments were removed, I found no statement about returning ("throwing") an error.

**Comment by KoltesDigital** [comment]
BTW, I saw a lot of message (aka generic) errors which could be formalized as: filter applied on an unexpected value type, filter expects an argument which wasn't set, filter expects an argument with a certain type but was another type, etc. How do you feel about adding these variants in Error?

**Comment by KoltesDigital** [comment]
Oops, too fast.

**Comment by Keats** [comment]
I don't think there's much point in differentiating runtime error types, that would just create way too many error types (there are tests and fns that would get those too to be consistent) to be useful.

**Comment by Keats** [comment]
Thanks for the PR!

**Pull request merged**